README.md

```diff
@@ -1 +1,134 @@
-# Code_Review
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [POrtfolio Finishing Mobile Version] <a name="about-project"></a>
+
+
+**Portfolio finishing Mobile Version** is a personal portfolio which describes all my skills and personal projects, you can go through and check it.
+
+## 🛠 Built With <a name="built-with"></a>
+
+    .Webhint
+    .Styleint
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **adding some emojis to the html file.**
+- **adding some more interesting background to webpage**
+- **adding more semantic tags to make it rwd**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites: have basic knowledge of html and css.
+
+In order to run this project you need:
+
+
+### Setup
+
+1) open your local terminal
+2) run git clone https:/github.com/sam2if/Portfolio_MV.
+
+### Install: vscode, eslint, node_js
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [samuel_lemma](https://github.com/sam2if)
+- Twitter: [samuel_lemma](https://twitter.com/samuellemma3733)
+- LinkedIn: [samuel_lemma](https://www.linkedin.com/in/samuel-lemma-7479b1214/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
```

Comment:
I think it would be good if you add the section Future Features, including 1-3 potential features to be added as it is the most important to describe what you are gonna improve.